### PR TITLE
add azure mongodb in terraform

### PR DESCRIPTION
**Pull request by pranaysahith**

### CODE REVIEW

File: deployment/terraform_azure/aks.tf [hunk 1]
```diff
@@ -73,3 +66,20 @@ resource "local_file" "aks_config" {
...
 }
+
+resource "kubernetes_namespace" "eventuate" {
+  metadata {
+    name = "eventuate-tram-examples-customers-and-orders"
+  }
+}
+
+resource "kubernetes_config_map" "mongo" {
+  metadata {
```
Comment (cer) [inline deployment/terraform_azure/aks.tf:77]: There needs to a YAML defining the config map for the non-AZ case.

File: deployment/terraform_azure/mongodb.tf [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      project = var.project
+      environment = var.env
+  }
+}
+
+resource "azurerm_cosmosdb_mongo_database" "default" {
+  name                = "customers_and_orders"
+  resource_group_name = azurerm_cosmosdb_account.db.resource_group_name
+  account_name        = azurerm_cosmosdb_account.db.name
+  throughput          = 400
```
Comment (cer) [inline deployment/terraform_azure/mongodb.tf:33]: I'd set this to a really low value.

File: deployment/terraform_azure/aks.tf (hunk 1 above)
Comment (pranaysahith) [reply deployment/terraform_azure/aks.tf:77]: Added config map file - deployment/kubernetes/secrets/mongodb-config.yaml
This needs to be deployed only on local k8s cluster.

File: deployment/terraform_azure/mongodb.tf (hunk 2 above)
Comment (pranaysahith) [reply deployment/terraform_azure/mongodb.tf:33]: 400 is the minimum allowed value.

File: deployment/terraform_azure/mongodb.tf [hunk 3]
```diff
@@ -0,0 +1,38 @@
+resource "azurerm_cosmosdb_account" "db" {
```
Comment (cer) [inline deployment/terraform_azure/mongodb.tf:1]: `db` is vague especially if there is more than one database. Please use a more descriptive name.

**Pull request merged**